docs/SPEC.md

```
@@ -71,13 +71,34 @@ Nested steps defined by H3 (`###`) headers.
 *   **Identifiers**: `### 1`, `### 1.1` (sequential), or `### Name` (named).
 *   **Aggregation**: Parent step outcome is derived from substeps via transitions (`ALL`/`ANY`).
 
-### 3.3 Runbooks
+### 3.3 Runbook List Shorthand
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)